srcpkgs/mesa/template

```diff
@@ -228,6 +228,16 @@ libgbm_package() {
 	}
 }
 
+libgbm-devel_package() {
+	short_desc="Mesa Generic buffer management API - development files"
+	depends="libgbm>=${version}_${revision}"
+	pkg_install() {
+		vmove /usr/include/gbm.h
+		vmove /usr/lib/libgbm.so
+		vmove /usr/lib/pkgconfig/gbm.pc
```

Comment:
Shouldn't use a leading slash in paths for `vmove`.

Reply:
done